modules/addons/coredns.tf

```diff
@@ -0,0 +1,12 @@
+resource "aws_eks_addon" "coredns" {
+  count = var.create_coredns_addon ? 1 : 0
+
+  cluster_name      = var.cluster_name
+  addon_name        = "coredns"
+  resolve_conflicts = "OVERWRITE"
+  addon_version     = lookup(var.coredns_versions, var.cluster_version, "Not Found")
```

Comment:
how AWS API will react when we pass `"Not Found"`, does there is validation on their end?

Reply:
This is on purpose, it will not even reach aws api. terraform plan will throw an error stating the addon version can not be `"Not Found"` giving a hint for the user that the cluster version is not supported by the module